webservice/src/main/resources/application.properties

```diff
@@ -13,3 +13,7 @@ lnp.lnd.certificatePath=tls.cert
 lnp.lnd.macaroonPath=admin.macaroon
 
 lnp.misc.ipRetrieverUrl=https://checkip.amazonaws.com/
+
+lnp.auth.expirationInMs=3600000
+lnp.auth.jwtSecret=L))::Z.&I!!vA_X.aR<*DScss&xfe.9d[I}{AnyQ+[#ik=W_4_y<?'uVtC@Jeg'j
+
```

Comment:
Czy $jwtSecret nie powinna zmieniać się co x czasu?

Reply:
Nie. Co w przypadku, kiedy zmieniamy secret, a użytkownicy są zalogowani i mają ważne tokeny? Gdybyśmy z tym szli na proda, to ten secret byłby w zmiennych środowiskowych, i można go ewentualnie zmieniać przy restarcie serwera.